org.eclipse.winery.workflowmodeler/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
```

Comment:
In my experience, TsLint is complaining about all the stars, which is why we agreed on using a normal comment block:
```
/**
 * Copyright
 * ...
 */
```

and for consistency, I would recommend to apply these changes to all files 🙂